biome.json

```diff
@@ -14,13 +14,12 @@
       "./coverage",
       "./.yarn",
       "./libs/gi/dm/GenshinData",
+      "./libs/gi/dm/HakushinData",
       "./libs/sr/dm/StarRailData",
       "./libs/zzz/dm/ZenlessData",
       "./libs/zzz/dm/HakushinData",
       "**/*_gen.json",
-      "./libs/gi/stats/Data",
-      "./libs/sr/stats/Data",
-      "./libs/zzz/stats/Data",
+      "./libs/gi/stats/Data/**",
```

Comment:
no longer ignoring data from sr/zzz?

Reply:
those ignores didn't work anyways (needs the **)
plus, we are actually formatting the files there as part of the executor, so we shouldn't ignore